pyls/plugins/jedi_completion.py

```diff
@@ -51,25 +51,34 @@ def pyls_completions(config, document, position):
 
     settings = config.plugin_settings('jedi_completion', document_path=document.path)
     should_include_params = settings.get('include_params')
-    include_params = (use_snippets(document, position) and
-                      snippet_support and should_include_params)
+    include_params = (snippet_support and should_include_params and
+                      use_snippets(document, position))
     return [_format_completion(d, include_params) for d in definitions] or None
 
 
+_IMPORTS = ('import_name', 'import_from')
```

Comment:
Please define this at the beginning of the file, before of after `_TYPE_MAP`

Reply:
Done